### PR TITLE
Upgrade to beta API

### PR DESCRIPTION
**Pull request by yuchaoran2011**

### CODE REVIEW

File: incubator/sparkoperator/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The following table lists the configurable parameters of the Spark operator char
 | Parameter                 | Description                                                  | Default                                |
 | ------------------------- | ------------------------------------------------------------ | -------------------------------------- |
 | `operatorImageName`       | The name of the operator image                               | `gcr.io/spark-operator/spark-operator` |
-| `operatorVersion`         | The version of the operator to install                       | `v2.4.0-v1alpha1-latest`               |
+| `operatorVersion`         | The version of the operator to install                       | `v2.4.0-v1beta1-latest`               |
 | `imagePullPolicy`         | Docker image pull policy                                     | `IfNotPresent`                         |
 | `sparkJobNamespace`       | K8s namespace where Spark jobs are to be deployed            | `default`                              |
```
Comment (liyinan926) [inline incubator/sparkoperator/README.md:29]: Can we change this from `default` to an empty string, which is the default in https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/main.go#L58?

**Comment by liyinan926** [comment]
Also looks like we should delete the row of `namespace` in the table at the bottom of https://github.com/helm/charts/blob/master/incubator/sparkoperator/README.md.

File: incubator/sparkoperator/README.md (hunk 1 above)
Comment (yuchaoran2011) [reply incubator/sparkoperator/README.md:29]: Ok. In `spark-rbac.yaml`, I fixed the condition. When sparkJobNamespace is empty string, the file is not applied.

**Pull request merged**